src/main/java/webs/hillel/Main.java

```diff
@@ -1,4 +1,46 @@
 package webs.hillel;
 
+import java.util.List;
+import java.util.logging.Logger;
+
 public class Main {
+    private static final Logger logger = Logger.getLogger(Main.class.getName());
```

Comment:
Please follow the naming conventions - if you create a constant (static final field of any type) then you should name it upper case, like this: `logger` -> `LOGGER`